Generator/Templates/GObject/class.sbntxt

```diff
@@ -67,6 +67,22 @@ namespace {{ namespace }}
         #endregion
 
         {{~ end ~}}
+        
+        #region Fields
+                
+        [StructLayout(LayoutKind.Sequential)]
+        protected {{'new'|get_if_parent}} struct Fields : IObjectStruct
+        {        
+{{ # TODO: Do we want to use field structs or put the fields in objects directly? 
+if (has_parent) && (name != "InitiallyUnowned") && (fields.size > 0)
+    fields[0].type.name = string.append fields[0].type.name ".Fields" 
+end}}
+{{~ for field in fields
+    include 'field.sbntxt' field
```

Comment:
The indentation is borken. To fix it it should probably look like the following and be indented at the correct level:

```
{{~ for $field in fields }}
{{ include 'signal.sbntxt' $field }}
{{ end ~}}
```

I think it is better to use `$field` instead of `field` in the loop as this is a local variable compared to a global one.

Reply:
I've managed to fix the indentation.

There's still a problem with spacing between each field being wildly inconsistent, not really sure how to go about fixing that.

Reply:
:+1:  I think it is fine for now. Those vertical white space problems have s.th. to do with the documentation if it is generated / available or not. It is happening for fields in structs, too (but with one line less in between...).

It's not very nice but probably involves some update to Scriban. I did not take the time to reproduce the exact steps involved in it as the behavior is probably intended but not usefull for our case. So we would need to create some minimal samples and explain why it is bad for us and make some suggestions on how to improve it to the scriban project.

I opened #160 to track this.